ctf/new.py

```diff
@@ -67,6 +66,9 @@ def new(
         )
         exit(code=1)
 
+    if template == Template.RUST_WEBSERVICE:
+        with_build_container = True
+
```

Comment:
I guess this could be simplified by assuming true if a build.yaml.j2 file is present in the template dir. However it can be a bit awkward to check with the current codebase